bin/protoc-go.sh

```diff
@@ -2,21 +2,21 @@
 
 set -eu
 
-go install ./vendor/github.com/golang/protobuf/protoc-gen-go
+gen() {
+    find controller/gen -name \*.pb.go -exec git rm -f '{}' +
```

Comment:
Why not just `git rm -r controller/gen`? Do we ever expect to have non-`pb.go` files in that directory?

Reply:
`git rm -rf` seemed scary to me. either approach is probably fine